### DISCUSSION TITLE
Wasted space, little usable space, maybe rethink UI for smaller screens?

### DISCUSSION THREAD

**Discussion by fcassata** [enchant97/note-mark#204] [Ideas]
I use small screens, 13" mostly, screenshot is the usual browser window I use. Only 37% of the default view is actual usable space (teal). Red is 100% wasted space since I can't adjust the width of the tree. 

No concrete ideas here, just wanted to point this out. I know I can click the two arrow to expand the editor area, but I want the tree visible too. It feels like this is just designed for huge screens.

<img width="1343" alt="sadf" src="https://github.com/user-attachments/assets/db0af8d0-136c-4c95-9b58-583599e0d82e">

**Comment by enchant97** [comment]
I don't really see the issue here, the screenshot shows plenty of line length for reading/editing notes. I think the note/book tree is a good width for common name lengths. And If more room is needed, the sidebar can be hidden.
